src/pricing/__tests__/PricingCard.js

```diff
@@ -6,10 +6,7 @@ import PricingCard from '../PricingCard';
 describe('PricingCard component', () => {
   it('should render without issues', () => {
     const component = shallow(
-      <PricingCard
-        info={['1 User', 'Basic Support', 'All Core Features']}
-        button={{ title: 'GET STARTED', icon: 'flight-takeoff' }}
-      />
+      <PricingCard button={{ title: 'GET STARTED', icon: 'flight-takeoff' }} />
```

Comment:
Instead of replacing this test, let's make a new test called `'it should render without info`

Reply:
okay got it @iRoachie